Makefile

```
@@ -224,6 +240,20 @@ images-machine-validation: ## Build the machine-validation runner + config image
 	done; \
 	docker buildx imagetools create -t $(IMAGE_REGISTRY)/machine-validation:$(IMAGE_TAG) $$tags
 
+images-machine-validation-arm: ## Build the native ARM64 machine-validation runner + config image
```

checkmake (0.3.2)

[warning] Target body for "images-machine-validation-arm" exceeds allowed length of 5 lines (12).

(maxbodylength)